tests/integration/kubernetes/gha-run.sh

```diff
@@ -420,6 +420,8 @@ function cleanup_kata_deploy() {
 function cleanup() {
 	platform="${1:-}"
 	test_type="${2:-k8s}"
+	# skip cleanup on CI for CRI-O due https://github.com/kata-containers/kata-containers/issues/10861
+	[[ "${CI:-}" = "true" && "${CONTAINER_RUNTIME:-}" = crio ]] && return
```

Comment:
> The cleanup is executed on latest step of the pipeline, and the environment is destroyed at the end anyway. I could have disabled it on the workflow yaml file too, but then to debug on CI environment late rit would be hard as changes to the workflow don't take effect on ones PR's jobs.

I suspect from the logs the hang to occur later, likely in `cleanup_kata_deploy`. Can you move this change there ?

Reply:
Just update this PR to address your comment @gkurz ! Thanks!